### ISSUE TITLE
Add the ability to force display buildings

### ISSUE THREAD

**Issue by JamesKingdom** [openstreetmap/iD#4574]
When trying to view some HOT buildings, some of them get hidden because there are too many of them - as shown below.
![](https://i.glideservers.com/u/jam90278f35-762d-403c-8267-26d50d9f7a73.jpg) 

Maybe we could add a way to manually load them - but warn the user that it could slow down their browser?

**Comment by bhousel** [comment]
I agree it would be nice to offer users more control in what gets shown/hidden.
